### PR TITLE
Show program reference in observation table

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: modules/web/client-model/src/main/scala/observe/ui/model/ObsSummary.scala [hunk 1]
```diff
@@ -38,7 +39,8 @@ case class ObsSummary(
   attachmentIds:      SortedSet[ObsAttachment.Id],
   observingMode:      Option[ObservingMode],
   visualizationTime:  Option[Instant],
-  posAngleConstraint: PosAngleConstraint
+  posAngleConstraint: PosAngleConstraint,
+  programReference:   Option[ProgramReference]
```
Comment (cquiroz) [inline modules/web/client-model/src/main/scala/observe/ui/model/ObsSummary.scala:43]: We are showing the program ref but if not present the proposal ref. Should we do the same here?

Comment (rpiaggio) [reply modules/web/client-model/src/main/scala/observe/ui/model/ObsSummary.scala:43]: I considered that, but are we actually going to get `READY` observations in observe without a program ref?

Comment (cquiroz) [reply modules/web/client-model/src/main/scala/observe/ui/model/ObsSummary.scala:43]: Can PIs change the obs status?

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**